raritan/context.py

```diff
@@ -124,6 +126,14 @@ def get_data_reference(self, name: str | list):
         bad_type = type(name)
         raise RuntimeError(f'Data references may only be gotten by string or list, {bad_type} provided.')
 
+    def print_all_data_references(context):
```

Comment:
This method could use a docstring. Otherwise, this is looking real good. Thanks @jacklchang!